### PR TITLE
docs: improve plugins' documentation (Chinese)

### PR DESCRIPTION
**Pull request by juzhiyuan**
### What this PR does / why we need it:

This PR aims to improve plugins' documentation in Chinese.

- [x] key-auth
- [x] jwt-auth
- [x] basic-auth
- [x] limit-conn
- [x] api-breaker
- [x] cors
- [x] limit-req
- [x] limit-count
- [x] referer-restriction
- [x] proxy-mirror

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/zh/latest/plugins/basic-auth.md [hunk 1]
```diff
@@ -21,128 +21,154 @@ title: basic-auth
...
 
-- [**名字**](#名字)
-- [**属性**](#属性)
-- [**如何启用**](#如何启用)
-- [**测试插件**](#测试插件)
-- [**禁用插件**](#禁用插件)
+启用该插件后，客户端访问路由、服务时需提供正确的用户名与密码，插件将从 HTTP 请求头 Authorization 中获取凭证信息。
 
-## 名字
+:::caution 注意
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/basic-auth.md:28]: https://docusaurus.io/docs/next/markdown-features/admonitions

File: docs/zh/latest/plugins/limit-conn.md [hunk 2]
```diff
@@ -21,62 +21,62 @@ title: limit-conn
...
-| key                | object  | required |        | ["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for", "consumer_name"] | 用户指定的限制并发级别的关键字，可以是客户端 IP 或服务端 IP。<br />例如，可以使用主机名（或服务器区域）作为关键字，以便限制每个主机名的并发性。 否则，我们也可以使用客户端地址作为关键字，这样我们就可以避免单个客户端用太多的并行连接或请求淹没我们的服务。 <br />当前接受的 key 有："remote_addr"（客户端 IP 地址）, "server_addr"（服务端 IP 地址）, 请求头中的"X-Forwarded-For" 或 "X-Real-IP", "consumer_name"（consumer 的 username）。 |
-| rejected_code      | string  | optional | 503    | [200,...,599]                                                                             | 当请求超过 `conn` + `burst` 这个阈值时，返回的 HTTP 状态码                                                                                                                                                                                                                                                                                                                                                                                    |
+## 参数
 
-**注：key 是可以被用户自定义的，只需要修改插件的一行代码即可完成。并没有在插件中放开是处于安全的考虑。**
```
Comment (Yiyiyimu) [inline docs/zh/latest/plugins/limit-conn.md:36]: Is there any reason to remove this line?

File: docs/zh/latest/plugins/key-auth.md [hunk 3]
```diff
@@ -21,129 +21,138 @@ title: key-auth
...
-    },
-    "upstream": {
-        "type": "roundrobin",
-        "nodes": {
-            "39.97.63.215:80": 1
-        }
+  "methods": ["GET"],
+  "uri": "/get",
+  "plugins": {
+    "key-auth": {}
```
Comment (Yiyiyimu) [inline docs/zh/latest/plugins/key-auth.md:148]: typo here

**Review by Yiyiyimu — APPROVED** [review]
mostly LGTM. Do we create another issue to fix EN doc according to this PR? We also could delete related screenshots after EN doc got fixed.

File: docs/zh/latest/plugins/limit-conn.md (hunk 2 above)
Comment (juzhiyuan) [reply docs/zh/latest/plugins/limit-conn.md:36]: I would prefer putting this line in FAQ or somewhere else 🤔 Our parameters have the most common values already.

File: docs/zh/latest/plugins/key-auth.md (hunk 3 above)
Comment (juzhiyuan) [reply docs/zh/latest/plugins/key-auth.md:148]: typo?

**Comment by juzhiyuan** [comment]
I see there have a wrong check 😳😳

image

File: docs/zh/latest/plugins/key-auth.md (hunk 3 above)
Comment (Yiyiyimu) [reply docs/zh/latest/plugins/key-auth.md:148]: need to remove it from plugin list 🤣

File: docs/zh/latest/plugins/limit-conn.md (hunk 2 above)
Comment (Yiyiyimu) [reply docs/zh/latest/plugins/limit-conn.md:36]: Yeah so it might be better to move it to FAQ in this PR or I guess we'll forget about it 🤣

**Comment by spacewander** [comment]
> 
> 
> I see there has a wrong check 😳😳 `Access-Control-Allow-Credentials`
> 
> image

The check is correct. You should change `Access-Controll-Max-Age`.

**Comment by juzhiyuan** [comment]
I use `Control` according to MDN[1], does this mean they are different?

[1] https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age

File: docs/zh/latest/plugins/key-auth.md (hunk 3 above)
Comment (juzhiyuan) [reply docs/zh/latest/plugins/key-auth.md:148]: ok 😂

**Comment by spacewander** [comment]
I mean you should change `Access-Controll-Max-Age` to `Access-Control-Max-Age`

**Comment by Yiyiyimu** [comment]
> Hi @Yiyiyimu, I have added some other plugins, this PR is ready for review. (10 Plugins in Chinese)

New docs LGTM 🚀

**Comment by juzhiyuan** [comment]
> I mean you should change `Access-Controll-Max-Age` to `Access-Control-Max-Age`

Done

**Comment by juzhiyuan** [comment]
For docs in English, I would prefer submitting a new PR ✌️

**Comment by iamayushdas** [comment]
> For docs in English, I would prefer submitting a new PR

okay i will be working on EN docs in next PR, will take help from @guoqqqi or @Yiyiyimu if i do wrong 
while understanding chinese

**Review by iamayushdas — APPROVED** [review]

**Comment by juzhiyuan** [comment]
@iamayushdas ok! Once this PR is merged without incorrect contents, then you could start doing it :)

**Comment by Yiyiyimu** [comment]
> okay i will be working on EN docs in next PR, will take help from @guoqqqi or @Yiyiyimu if i do wrong
> while understanding chinese

Hi @iamayushdas Thanks for the volunteer! But as I told you before, it might not be a good idea to do the Chinese-English translation for a non-CNEN-bilingual speaker, since it is not a good idea to fully rely on translation tools to do the job. I think we still need a Chinese contributor on this kind of task.

**Comment by juzhiyuan** [comment]
After discuss with @Yiyiyimu, I agree with him, this is more challenge for a non-CN speaker, you could pick other issues like `Web Test` or `Featrures` or `bugfix` issues.

File: docs/zh/latest/plugins/api-breaker.md [hunk 4]
```diff
@@ -21,98 +21,69 @@ title: api-breaker
...
-- [**禁用插件**](#禁用插件)
+启用该插件后，网关将根据配置判断上游是否异常，若异常，则直接返回预设的错误码，且在一定时间内不再访问上游。
 
-## 定义
+## 参数
 
-该插件实现 API 熔断功能，帮助我们保护上游业务服务。
+|        参数名称         |    类型    | 必选  | 默认值 | 使用范围  |                                        描述                                        |
+| :---------------------: | :--------: | :---: | :----: | :-------: | :--------------------------------------------------------------------------------: |
+|   break_response_code   |   整数型   |  是   |        | 200 ~ 599 |                           上游不健康时，将返回该状态码。                           |
```
Comment (spacewander) [inline docs/zh/latest/plugins/api-breaker.md:32]: The indentation should be left alignment. The center alignment can't be maintained by humans. Please fix all the similar places.

File: docs/zh/latest/plugins/cors.md [hunk 5]
```diff
@@ -21,85 +21,99 @@ title: cors
...
 ```
 
-## 测试插件
+接着，访问路由进行测试：
+
+```bash
+# 场景1：在未启用 CORS 插件时，访问资源：
+
+## Request
+$ curl -i http://127.0.0.1:9080/get -v
```
Comment (spacewander) [inline docs/zh/latest/plugins/cors.md:72]: Would be better if we can provide a refined example without `-v`.

File: docs/zh/latest/plugins/basic-auth.md (hunk 1 above)
Comment (spacewander) [reply docs/zh/latest/plugins/basic-auth.md:28]: Is it necessary to use docusaurus special syntax? Why not use `****` directly?

**Review by spacewander — CHANGES_REQUESTED** [review]

**Comment by moonming** [comment]
> For docs in English, I would prefer submitting a new PR ✌️

English doc first

**Review by moonming — CHANGES_REQUESTED** [review]
Please fix English doc first

File: docs/zh/latest/plugins/basic-auth.md (hunk 1 above)
Comment (juzhiyuan) [reply docs/zh/latest/plugins/basic-auth.md:28]: Those contents are used in the docs site, this syntax will be rendered as `Admonitions Box`.

image

File: docs/zh/latest/plugins/api-breaker.md (hunk 4 above)
Comment (juzhiyuan) [reply docs/zh/latest/plugins/api-breaker.md:32]: 🤔 ok

**Comment by juzhiyuan** [comment]
> Is it necessary to use docusaurus special syntax? Why not use **** directly?

Those contents are used in the docs site, this syntax will be rendered as `Admonitions Box`.

image

**Comment by juzhiyuan** [comment]
> Please fix English doc first

@moonming Please review Chinese docs first.

File: docs/zh/latest/plugins/api-breaker.md (hunk 4 above)
Comment (juzhiyuan) [reply docs/zh/latest/plugins/api-breaker.md]: done

**Comment by juzhiyuan** [comment]
After reviewing Chinese versions, I will submit PRs to update EN.

**Review by membphis — CHANGES_REQUESTED** [review]
I recommend one PR to fix one type of problem, rather than putting different problems together in one PR.
This will make it difficult to review and synchronize code later. 

For example, if we want to change `目录` -> `简介`, we can do this in a separate PR. It is an easier way to review and merge.

**Comment by membphis** [comment]
split this PR into smaller ones are better

image

**Comment by juzhiyuan** [comment]
Hi, to prevent from misleading PRs or contents, I will split them into smaller PRs.

**Comment by juzhiyuan** [comment]
Each Plugin PR will have both Chinese & English changes.

**Pull request closed without merging**